### PR TITLE
Add PublishToAzureServiceBusV2 task

### PR DESCRIPTION
**Pull request by DenisNikulin5**
**Task name**: PublishToAzureServiceBusV2

**Description**: Added the new version of task to use AzureRM endpoints

**Documentation changes required:** Y

**Added unit tests:** N

**Attached related issue:** [AB#2152974](https://dev.azure.com/mseng/b924d696-3eae-4116-8443-9a18392d8544/_workitems/edit/2152974)

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/PublishToAzureServiceBusV2/task.json [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        "Build",
+        "Release"
+    ],
+    "runsOn": [
+        "Server"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 2,
+        "Minor": 239,
```
Comment (starkmsu) [inline Tasks/PublishToAzureServiceBusV2/task.json:19]: Let's use 240 here

File: Tasks/PublishToAzureServiceBusV2/task.json [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        {
+            "name": "serviceBusQueueName",
+            "type": "string",
+            "label": "ms-resource:loc.input.label.serviceBusQueueName",
+            "required": true,
+            "defaultValue": "",
+            "helpMarkDown": "ms-resource:loc.input.help.serviceBusQueueName"
+        },
+        {
+        "name": "serviceBusNamespace",
```
Comment (starkmsu) [inline Tasks/PublishToAzureServiceBusV2/task.json:50]: Please fix code indentation

File: Tasks/PublishToAzureServiceBusV2/task.json [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+        "type": "string",
+        "label": "ms-resource:loc.input.label.serviceBusQueueName",
```
Comment (starkmsu) [inline Tasks/PublishToAzureServiceBusV2/task.json:52]: Copy-paste error - should be 'serviceBusNamespace', right?

File: Tasks/PublishToAzureServiceBusV2/task.json [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+        "required": true,
+        "defaultValue": "",
+        "helpMarkDown": "ms-resource:loc.input.help.serviceBusQueueName"
```
Comment (starkmsu) [inline Tasks/PublishToAzureServiceBusV2/task.json:55]: Another copy-paste error

File: Tasks/PublishToAzureServiceBusV2/task.json (hunk 1 above)
Comment (DenisNikulin5) [reply Tasks/PublishToAzureServiceBusV2/task.json]: Fixed

File: Tasks/PublishToAzureServiceBusV2/task.json (hunk 3 above)
Comment (DenisNikulin5) [reply Tasks/PublishToAzureServiceBusV2/task.json]: Fixed

File: Tasks/PublishToAzureServiceBusV2/task.json (hunk 4 above)
Comment (DenisNikulin5) [reply Tasks/PublishToAzureServiceBusV2/task.json]: Fixed

**Review by manolerazvan — APPROVED** [review]

**Pull request merged**